GroundDBMemoryPool/src/include/rdma_client.hh

```diff
@@ -29,4 +29,19 @@ namespace mempool
         const size_t rofs = 0 /* offset in remote memory region */
     );
 
+    void init_pat_access_on_client(
+        struct resources* res,
+        struct request_buffer* pat_req_buf,
+        const char *server_name, /* server host name */
+        const int tcp_port,     /* server TCP port */
+        const int ib_port       /* server IB port */
+    );
+    // Get the page's remote address and LSN, -1 if not found
+    std::pair<uintptr_t, uint64_t> get_page_address_from_mempool(
+        const struct resources *res,
+        const struct memory_region *mr,
+        const struct connection *conn,
+        KeyType page_id
+    );
```

Comment:
Return value is `uintptr_t` in the implementation. Is this interntional?

Reply:
I think both `uintptr_t` and `uint64_t` are acceptable.

Reply:
But implementation's function prototype is 

 ```
uintptr_t get_page_address_from_mempool(
        const struct resources *res,
        const struct memory_region *mr,
        const struct connection *conn,
        KeyType page_id
)
```

instead of being `std::pair<uintptr_t, uint64_t> get_page_address_from_mempool`. Implementation doesn't return a pair.

Reply:
The second element in the pair is the LSN of the page, which is needed as well when accessing PAT.

Reply:
I may miss how `std::pair` works, but on line 98 of connect.cc in the PR, `get_page_address_from_mempool ` only returns one single value instead of a pair of values.

Reply:
Sorry i forgot to commit the updates of `connect.cc`